unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Passport/Passport/PlayerInfo/PassportPlayerInfoComponentView.cs

```diff
@@ -113,20 +133,28 @@ private void RemoveFriendsFocused(bool isFocused)
             unfriendVariation.SetActive(isFocused);
         }
 
-        private void BlockFriendFocused(bool isFocused)
-        {
-            alreadyBlockedVariation.SetActive(!isFocused);
-            unblockVariation.SetActive(isFocused);
-        }
-
         private void SetName(string name)
         {
-            this.name.text = name;
+            if (name.Contains('#'))
+            {
+                this.name.SetText(name.Split('#')[0]);
+                address.SetText($"#{name.Split('#')[1]}");
+            }
+            else
+            {
+                this.name.SetText(name);
+                address.SetText("");
+            }
+            LayoutRebuilder.ForceRebuildLayoutImmediate(usernameRect);
```

Comment:
Is this `ForceRebuildLayoutImmediate` really needed? In case it is, you should use `Utils.ForceRebuildLayoutImmediate(usernameRect)` that is more performant.